content/guides/models/track/runs/_index.md

```diff
@@ -168,17 +168,20 @@ run.finish()
 ```
 {{% /alert %}}
 
+If you [group runs]({{< relref "grouping.md" >}}) into experiments, you can move a run into or out of a group or from one group to another.
+
 Each run has a state that describes the current status of the run. See [Run states]({{< relref "#run-states" >}}) for a full list of possible run states.
 
 ## Run states
 The proceeding table describes the possible states a run can be in: 
 
 | State | Description |
 | ----- | ----- |
-| Finished| Run ended and fully synced data, or called `wandb.finish()` |
-| Failed | Run ended with a non-zero exit status | 
-| Crashed | Run stopped sending heartbeats in the internal process, which can happen if the machine crashes | 
-| Running | Run is still running and has recently sent a heartbeat  |
+| `Crashed` | Run stopped sending heartbeats in the internal process, which can happen if the machine crashes. | 
```

Comment:
Summoning @kptkin  to review this portion.

Reply:
Mostly I just sorted this list alphabetically rather than by whatever the rubric was before.